src/js/main.js

```diff
@@ -485,9 +485,25 @@ require([
             jquery(".content").removeClass("btn-primary");
             jquery(this).addClass("btn-primary");
             jquery(this).removeClass("btn-info");
+
+            /*
+             * Determine if /data should be returned for an item
+             *  based on the Sharing API name key value for that
+             *  item.  Looks for a period in the name to indicate
+             *  that it is a file based item, e.g. roads.kmz.
+             */
+            var checkData = function(id, name){
```

Comment:
Instead of the code you have, which leverages Promise (not supported in IE), how about using jquerry.Deferred:
```
var checkData = function(id, name){
    var deferred = new jquery.Deferred();
    if ((!name) || ((name) && (name.indexOf('.') < 0))) {
        deferred.resolve(portal.itemData(id));
    }
    else { deferred.resolve(null); }
    return deferred;
}; 
```

Reply:
That would be a great solution (temporary or long term). I forgot those are in jquery. 
I've got the new tooling in place that runs `main.js` through babel so we don't need to lean on jquery to solve this particular problem. It might be nice to unshackle this entire project from jquery over the long term.